plugins/Proxy/Controller.php

```diff
@@ -26,6 +27,15 @@ class Controller extends \Piwik\Plugin\Controller
     const JS_MIME_TYPE = "application/javascript; charset=UTF-8";
 
     /**
+     * Proxy method to call API methods using current session for authentication
+     */
+    public function callApi()
```

Comment:
Do you need this comment above? So maybe would be better to use more meaningful name eg. ```callApiWithCurrentSession```